src/nova-components/CreateFavorites/resources/js/components/DropdownField.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="field-container">
+    <div class="label-spacing">
+      <Label :label="label" :isRequired="true" />
+    </div>
+    <div class="input-spacing">
+      <select
+        :class="{
+          'input-error': error,
+          'input-decoration': true,
+          'arrow-decoration': true,
+        }"
+        v-model="input"
+        @change="$emit('changeInput', $data.input, label)"
```

Comment:
don't use events pass function as a prop to component then call this function inside component